editor.planx.uk/src/ui/editor/SortControl/utils.ts

```diff
@@ -3,6 +3,8 @@ import { slugify } from "utils";
 import { z } from "zod";
 
 import { SortableFields } from "./SortControl";
+import { get } from "lodash";
```

Comment:
nit: when using a single lodash function we typically favor `import get from "lodash/get"` to import just the required module and not the whole package